x-pack/plugins/alerting/server/task_runner/transform_action_params.ts

```diff
@@ -115,14 +117,19 @@ export function transformActionParams({
       flapping,
       consecutiveMatches,
     },
+  };
+
+  const variablesWithADDFields: Record<string, unknown> = {
```

Comment:
I think it's a typo: should be `variablesWithAADFields`, right?

Reply:
Ty!